benchmark_apis/c_api_driver/BUILD_GUIDE.md

```diff
@@ -0,0 +1,40 @@
+# Build Guide for cuOpt JSON to C API
```

Comment:
Do we need to build c_api_driver or it will get built as part of benchmark script ?